shared/Microsoft.Extensions.PropertyHelper.Sources/PropertyHelper.cs

```diff
@@ -121,20 +121,20 @@ public void SetValue(object instance, object value)
         /// Creates and caches fast property helpers that expose getters for every public get property on the
         /// underlying type.
         /// </summary>
-        /// <param name="instance">the instance to extract property accessors for.</param>
-        /// <returns>a cached array of all public property getters from the underlying type of target instance.
+        /// <param name="typeInfo">The type info to extract property accessors for.</param>
+        /// <returns>A cached array of all public property getters from the underlying type of target instance.
```

Comment:
> of all public properties of the specified type. 

(no instance anymore). That said, I'm ok if you removed the docs entirely too. It's a shared src type and doesn't need elaborate doc comments